<?xml version="1.0"?>
<launch>
  <!-- ROSCORE -->
	<master auto="start"/>
  <!--  Stage Simulator -->
  	<node pkg="stage_ros" type="stageros" name="stageros" args="$(find swarm_stage_ros)/launch/custom.world" respawn="true" output="log"/> 
        <node name="robot0" pkg="swarm_stage_ros" type="posctrl" args="robot_0" output="screen"/>
        <node name="robot1" pkg="swarm_stage_ros" type="posctrl" args="robot_1" output="screen"/>
        <node name="robot2" pkg="swarm_stage_ros" type="posctrl" args="robot_2" output="screen"/>
        <node name="controller" pkg="swarm_stage_ros" type="controller" args="3" output="screen"/>
</launch>